### DISCUSSION TITLE
Daily Standup and General Meetings

### DISCUSSION THREAD

**Discussion by joshuafuller** [danielmiessler/Fabric#545] [Polls]
Hi everyone,

I hope you're all doing well. I wanted to propose adding two new patterns to the Fabric project: one for Daily Standup meetings and another for more General Meetings. These patterns aim to streamline and enhance the efficiency of our meeting routines using AI.

#### Context
I currently use Whisperx to transcribe every Standup and almost every other internal meeting. The main purpose of this is to transcribe notes and then feed them into Obsidian. This creates a somewhat cobbled-together workflow that I believe could be significantly improved.

Initially, I was working on my own Python script to manage this process. However, I realized that using Fabric, which I already use, could be a better approach. Therefore, I'm considering refining the System Prompts for more general-purpose use within Fabric.

I've noticed much better results when using two separate prompts, one for each type of meeting. I'd love to hear others' thoughts on this approach.

#### Daily Standup Pattern
- Facilitates daily standup meetings by automating the collection of updates from team members.
- Summarizes key points and action items.
- Ensures everyone is aligned and aware of their tasks.

#### General Meeting Pattern
- Designed for various meeting types (e.g., project updates, brainstorming sessions).
- Organizes discussion points, decisions, and action items.
- Provides a summary of the meeting that can be shared with attendees.

#### Additional Considerations
- I specifically use Whisperx with diarization to help pick out speakers.
- LLMs are usually adept at identifying names mentioned during the meeting and properly attributing attendance.
- Date and time injection and format still seem to be inconsistent.

I have two patterns that I am happy to share, but I think they need to be more generalized before adding to the repo. Currently, they are pretty tailored to my meetings.

I'm not a traditional developer and am having a hard time generalizing what others might want in these system prompts. However, I believe these additions could be valuable for many users. I'm excited to hear your thoughts and see if there's interest in developing these patterns. Please vote in the poll below!

**Comment by JerichoCruz** [comment]
Hi Joshua, I'd love to see the patterns you mentioned!

**Comment by criadoperez** [comment]
Looking forward to see this feature. 
Still planing to develop it?
